util/logutil/log.go

```diff
@@ -345,3 +347,30 @@ func SetLevel(level string) error {
 	zaplog.SetLevel(l.Level())
 	return nil
 }
+
+type correlationIDType int
+
+const (
+	ctxLogKey correlationIDType = iota
+)
+
+// Logger gets a contextual logger from current context.
+// contextual logger will output common fields from context.
+func Logger(ctx context.Context) *zap.Logger {
+	if ctxlogger, ok := ctx.Value(ctxLogKey).(*zap.Logger); ok {
+		return ctxlogger
+	}
+	return plog.L()
+}
+
+// WithConnID attaches connId to context.
+func WithConnID(ctx context.Context, connID uint32) context.Context {
+	return context.WithValue(ctx, ctxLogKey, plog.L().With(zap.Uint32("connID", connID)))
```

Comment:
s/connID/conn/?

Reply:
As discussed yesterday, it's better to extract these common used keywords to some variables?